tap.go

```diff
@@ -39,7 +39,10 @@ func (bucket *WalrusBucket) StartTapFeed(args sgbucket.FeedArguments, dbStats *e
 		bucket.lock.Unlock()
 	}
 
-	go feed.run()
+	go func() {
+		close(args.DoneChan)
+		feed.run()
```

Comment:
Doesn't this need to be closed _after_ `feed.run()`?

Reply:
Yeah, that is correct. Changes are included in the latest commit.